test/resources/microshift-rpm.resource

```diff
@@ -27,12 +27,26 @@ Install MicroShift RPM Packages From System Repo
 
 Install MicroShift RPM Packages From Repo
     [Documentation]    Installs MicroShift RPM packages from the specified URL
-    [Arguments]    ${repo_url}    ${version}
-    Configure MicroShift Repository    ${repo_url}
-    Install MicroShift RPM Packages From System Repo    ${version}
-    Unconfigure MicroShift Repository
+    [Arguments]    ${repo}    ${version}
 
-Configure MicroShift Repository
+    IF    "http" in "${repo}"
```

Comment:
Quoting can be weird in RF. You might not need quotes around the variable here.

The other way to solve this might be to do more of the work to set up the repos from the scenario script, which might be easier because it's straight bash. For example, if that script logged in and enabled both repos, and then passed very detailed version info into the test, the test itself would be "install old version; verify it works; install new version; verify it works" without all of the repo manipulation.

Reply:
If you want to take that approach, we could just disable this scenario to remove some of the time pressure to get it working.

Reply:
I think I had issues when missed the quotes - I think python didn't interpret that as a string type.

As for the idea - sounds good, but isn't it more like refactor than fix? We have all the tools already in the RF, so to me it would be a candidate for separate PR.

Reply:
I didn't notice this was the 4.15 branch. We could also just hard-code repo names here, since they aren't going to change. Then we could work on making the main branch smarter so we don't have to keep updating those settings over time.

And yes, what I proposed would have been a refactor/rewrite, that's why I suggested temporarily disabling the scenario in 1 PR and doing the other work in another.

Reply:
On the second thought - that might be a better idea. I think this problem shows that we should keep the repos of the hypervisor and test harness separate, so they are explicitly enabled and used rather than having luck